embedded/app/main/badgerloop_HV.cpp

```diff
@@ -58,8 +60,9 @@ printf("Ezra\n");
 	struct sigaction sig;
     sig.sa_sigaction = emergQuitter;
     sigaction(SIGINT, &sig, NULL);
+
     /* Start 'black box' data saving */
-/*    SetupDataDump();*/
+    SetupDataDump();
```

Comment:
If data dump never ended up working, we might want to leave it commented out in master